app/views.py

```diff
@@ -87,8 +87,19 @@ def login():
             return "There was an error.", 500
         # Else success, redirect to index page
         else:
-            session['username'] = username
             session['token'] = token['token']
+            user = getUser(username)
+
+            if type(user) is list:
```

Comment:
@ElijahCaine @subnomo Then this is another check that should be put inside `get_user()`?